debian/asterisk.prerm

```diff
@@ -0,0 +1,8 @@
+#! /bin/sh
+
+set -e
+
+rmdir /usr/local/share/asterisk/sounds 2>/dev/null || true
```

Comment:
You can ignore this comment if you want
`rm --ignore-fail-on-non-empty /usr/local/share/asterisk/sounds`

Reply:
That was taken from the original Debian packaging: https://salsa.debian.org/pkg-voip-team/asterisk/-/blob/debian/latest/debian/asterisk.prerm. I think we'll avoid diverging from them if not necessary.